README.md

```diff
@@ -1,20 +1,20 @@
 # Winton Kafka Streams
 
-Implementation of [Apache Kafka Streams](https://kafka.apache.org/documentation/streams/) in Python. 
+Implementation of [Apache Kafka's streams API](https://kafka.apache.org/documentation/streams/) in Python. 
 
 ## What and why?
 Apache Kafka is an open-source stream processing platform developed
 by the Apache Software Foundation written in Scala and Java. Kafka
-Streams is a Java library for building distributed stream processing
-applications using Apache Kafka. Kafka Streams does not require any
+has streams API added for building stream processing applications
```

Comment:
https://github.com/wintoncode/winton-kafka-streams/pull/5

Above PR should fix the case.